docs/README.md

```diff
@@ -0,0 +1,59 @@
+## Space API Documentation
+
+Contents
+
+- [Usage](#usage)
+- [Class Definitions](class-definitions.md)
+  - [Spaces](class-definitions.md#spaces)
+  - [Space](class-definitions.md#space)
+  - [SpaceMember](class-definitions.md#spacemember)
+
+### Usage
+
+To use the Space API, you'll need to have an Ably client already set up.
+Here's an example of how you can create a new instance of the client and pass it to the Spaces constructor:
+
+```ts
+import { Realtime } from 'ably/promise';
+import Spaces from '@ably-labs/spaces';
+
+const client = new Realtime(process.env.ABLY_KEY);
+const spaces = new Spaces(client);
```

Comment:
Will this change if we go with Option 3 in the DR?

Reply:
Yes, if we wish to recommend passing the options instead of an ably client we will have to reflect those changes here